docs/mycoind/setup.rst

```diff
@@ -10,30 +10,34 @@ for both go and node.
 **WARNING**
 
 This is only tested under Linux and OSX.
-It will most likely not work under Windows, even with Cygwin.
-If you use windows, please make a PR with any adjustments so
-this tutorial works for Windows as well.
+If you want to run under windows, the only supported *development* environment
+is using WSL (Windows Subsytem for Linux) under Windows 10.
+Follow `these directions <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__
+to setup ubuntu in WSL, then try the rest in your Ubuntu shell
 
 Install Go
 ==========
 
-You will need to have the go tooling installed, version 1.9+.
+You will need to have the go tooling installed, version 1.11.4+ (or 1.12).
 If you do not already have it, please
 `download <https://golang.org/dl/>`_ and
 `follow the instructions <https://golang.org/doc/install>`__
 from the official Go language homepage. Make sure to read down
 to `Test Your Installation <https://golang.org/doc/install#testing>`__.
+(Note this is not included in Ubuntu apt tooling until 19.04)
 
 We assume a standard setup in the Makefiles, especially to
-build tendermint nicely. That means you must set up `GOPATH`,
-you must check out all source code under `$GOPATH/src`,
-and you must add the default install directory to your `PATH`.
+build tendermint nicely. That means you must set up ``GOPATH``,
```

Comment:
I believe since we use Go modules we no longer want to use `GOPATH`. We do not want to use it, because it is one of the requirements to use Go modules.